resolwe/flow/managers/workload_connectors/kubernetes.py

```diff
@@ -860,6 +863,32 @@ def start(self, data: Data, listener_connection: Tuple[str, str, str]):
             "It took {:.2f}s to send config to kubernetes".format(end_time - start_time)
         )
 
+        # Patch the PVC-s with owner references to the job.
+        patch = []
+        for claim_name in created_claim_names:
```

Comment:
This is wrong, since it will add two copies of the same ownerReference. You only need a list containing one Job owner reference and apply it to each PVC claim.

Reply:
To clarify, the list has to remain, since ownerReferences expects a list. The object in the kubectl yaml would look like this:
```
  ownerReferences:
  - apiVersion: batch/v1
    kind: Job
    name: job-name
    uid: some-uid
```

Reply:
Done.